source/tests/system/nirfmxbt_driver_api_tests.cpp

```diff
@@ -0,0 +1,80 @@
+#include <gtest/gtest.h>
+
+#include "device_server.h"
+#include "niRFmxBT.h"
+#include "nirfmxbt/nirfmxbt_client.h"
+
+using namespace ::testing;
+using namespace nirfmxbt_grpc;
+namespace client = nirfmxbt_grpc::experimental::client;
+
+namespace ni {
+namespace tests {
+namespace system {
+
```

Comment:
Try to remember to put an anonymous namespace around the test code. 

```cpp
namespace ni {
namespace tests {
namespace system {
namespace {
```

Otherwise I think it can cause problems with functions with the same name.

Reply:
I've implemented this suggestion.